rclcpp/include/rclcpp/executors.hpp

```diff
@@ -67,31 +111,31 @@ using rclcpp::executors::SingleThreadedExecutor;
  * \return The return code, one of `SUCCESS`, `INTERRUPTED`, or `TIMEOUT`.
  */
 template<typename FutureT, typename TimeRepT = int64_t, typename TimeT = std::milli>
+[[deprecated("use spin_node_until_complete(Executor &,"
+                                           "node_interfaces::NodeBaseInterface::SharedPtr,"
+                                           "const Condition &, DurationT) instead")]]
```

Comment:
nitpick: Typically we do not match indent like this, we indent everything or nothing, here's my suggestion:

```suggestion
[[deprecated(
  "use spin_node_until_complete(Executor &, node_interfaces::NodeBaseInterface::SharedPtr, "
  "const Condition &, DurationT) instead"
)]]
```

Any easy rule of thumb is "only indent a multiple of 2 (or 4 for Python), and never more than a single indent if you can help it".

This indents 43 spaces a weird, odd-numbered number of spaces, and wastes a lot of space for no reason.

Also consider if you changed the name of the function, you'd have to adjust the indentation of all of the lines to match, making future diffs more difficult.

Also, your line breaks lost a separating single whitespace between arguments that should be there.